### PR TITLE
Feat#11_create-readme

### PR DESCRIPTION
**Pull request by CasuallyCaffeinated**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,14 @@
...
+### Built by the Community, for the Community.
+![React Logo](https://img.shields.io/badge/React-20232A?style=for-the-badge&logo=react&logoColor=61DAFB) ![TypeScript](https://img.shields.io/badge/TypeScript-007ACC?style=for-the-badge&logo=typescript&logoColor=white) ![Syled Components](https://img.shields.io/badge/styled--components-DB7093?style=for-the-badge&logo=styled-components&logoColor=white) ![Storybook Logo](https://i.imgur.com/1wo0uBy.png)
+***
+### Getting started:
+***
+### 1. Use your preferred method to clone the repository.
+* ### E.g. `git clone https://github.com/open-code-academy/community-ui.git` 
+### 2. Run `npm install` to pull in all of the dependencies defined in a package.json file and generate a node_modules folder with the installed modules.
+
+### 3. Run the command `npm run develop` in order to start up your Storybook server.
```
Comment (aetheode) [inline README.md:12]: I think we should actually switch to using yarn v1 (ie yarn install && yarn develop would be the commands here)

I know that we already have a package-lock.json, but I tend to prefer yarn over npm as it provides concurrent downloads of packages, thus anyone new to the project can get started faster. That being said, I am open to any suggestion on this matter

Comment (CasuallyCaffeinated) [reply README.md:12]: We can certainly discuss this. I've never used Yarn and have no experience with it, but if everyone deems it the right way to go, we can definitely go in that direction. 

Although, I must ask, would it not be possible to have both an npm build and a yarn build? I've seen many projects in the past offer either option and allow a developer to choose. I may be talking out of place, but I am curious.

Comment (CasuallyCaffeinated) [reply README.md:12]: I'd also like to mention that there is a known issue with Husky when using Yarn on Windows machines: https://typicode.github.io/husky/#/?id=yarn-on-windows 

So if we do switch over to Yarn, we'll need to keep this in mind for people using Windows machines.

Comment (aetheode) [reply README.md:12]: True but luckily an easy workaround exists

Comment (aetheode) [reply README.md:12]: Here is an interesting article on the topic - https://dev.to/samithawijesekara/the-difference-between-npm-and-yarn-2j3p

Likely this should be spun off into a SPIKE - just wanted to mention it here when I saw it, but it is a bit off-topic for this PR

Comment (OleDakotaJoe) [reply README.md:12]: I'd certainly say there could be some advantage to using yarn. Though, there is also certainly no issue at all with using npm. 
Let's drop a spike in the issues, and discuss it further .

**Pull request merged**